src/scripts/main.js

```diff
@@ -1,3 +1,18 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+for (let i = 0; i < table.rows.length; i++) {
+  const row = table.rows[i];
+  const cells = row.cells[1].innerHTML;
+  const newCells = row.insertCell(row.cells.length - 1);
```

Comment:
Bug: The `insertCell()` method inserts a new cell into the current row at a specified position, and returns a reference to the new cell. However, you're not using this reference to do anything. Instead, you're directly assigning to `newCells.outerHTML`, which effectively discards the cell you just inserted. You should instead use the returned cell reference to set the inner HTML of the new cell.